SwipeDirection: -1

SwipeVelocity: 0.15

OrthogonalDamping: 0.8

OrthVelLim: 0.25

SwipeTarget: [-0.5 ,-0.3 , 0.15]
